cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```

Comment:
Fixtures are mocked data that you can setup in e2e testing that represents common data bits, think PageObjects if you are from that space

Reply:
Or alias to methods on APIS